physx/snippets/snippetloadcollection/SnippetLoadCollection.cpp

```diff
@@ -417,9 +417,12 @@ int snippetMain(int argc, const char *const* argv)
 
 	if (firstCollection)
 		firstCollection->release();
-
-	gScene->simulate(1.0f/60.0f);
-	gScene->fetchResults(true);
+	
+	for( PxU32 j = 0; j < 50; J++)
```

Comment:
@BenGood This is an endless loop at best, or a compile-time error - undeclared identifier at worst.

Reply:
right - this is why I should push tested changes properly and not copy them into the github's web interface. 

I just wanted to make something available for anyone who finds themselves as confused as I was because I don't think the snippet actually works as it is.